moveit_ros/moveit_servo/test/pose_tracking_test.cpp

```diff
@@ -135,15 +176,15 @@ TEST_F(PoseTrackingFixture, OutgoingMsgTest)
     size_t msg_count = 0;
     while (++msg_count < 100)
     {
-      target_pose_pub_.publish(target_pose);
-      ros::Duration(0.01).sleep();
+      target_pose_pub_->publish(target_pose);
+      std::this_thread::sleep_for(std::chrono::milliseconds(10));
```

Comment:
This should be done with a rate instead of a sleep.

Reply:
I agree!